core/src/main/java/edu/ucr/cs/riple/core/metadata/field/FieldDeclarationAnalysis.java

```diff
@@ -15,17 +15,33 @@
 import java.util.Collections;
 import java.util.Optional;
 import java.util.Set;
-import java.util.stream.Collectors;
 
+/**
+ * Detects multiple inline field declarations. An annotation will be injected on top of the field
+ * declaration statement, and in inline multiple field declarations that annotation will be
+ * considered for all declaring fields. This class is used to detect these cases and adjust the
+ * suggested fix instances. (e.g. If we have Object f, i, j; and a Fix suggesting f to be {@code
+ * Nullable}, this class will replace that fix with a fix suggesting f, i, and j be {@code
+ * Nullable}.)
```

Comment:
Yes, we discussed this earlier that there are two alternatives here, one is to break it into a single field declaration or keep the current approach. Since the existing annotations on the field declaration may not be something we can easily repeat it, safe way for now is to consider the annotation for all fields as NullAway does.

 ```java
@Annot(Param1, Param2) Object f = value, i, j;
```
May not make sense if we just break it to below:
```java
@Annot(Param1, Param2) Object f = value;
@Annot(Param1, Param2) Object i, j;
```

Reply:
Ah, right, makes sense. Seems like a bit of an unfortunate limitation in some cases, but I see why it would be hard to split the field declarations and still guarantee correctness in general in the presence of arbitrary annotations. Might be worth a quick check on Uber's internal codebase linting against declaring multiple fields in the same line.